packages/grid/_modules_/grid/models/params/gridCellParams.ts

```diff
@@ -18,15 +18,15 @@ export interface GridCellParams {
   /**
    * The cell value, but if the column has valueGetter, use getValue.
    */
-  value: GridCellValue;
+  value: T;
   /**
    * The cell value formatted with the column valueFormatter.
    */
-  formattedValue: GridCellValue;
+  formattedValue: T;
```

Comment:
I don't think the formattedValue has always the same type as the value

Reply:
https://github.com/final-form/react-final-form/blob/2d9e7129f40efd39450c1d581956e8cfce17a20a/typescript/index.d.ts#L90

Final form decides not to care about the formatted value in its interface

Another option would be to do `GridCellParams<T = GridCellValue, FormattedT = T, RowType = any>`
But I'm not sure it's worth it

Reply:
Yes, the formatted value can have other type than `T`. I agree with adding one more generic, like @flaviendelangle suggested.